e2e/test/scenarios/collections/instance-analytics.cy.spec.js

```diff
@@ -142,5 +144,86 @@ describeEE("scenarios > Instance Analytics Collection", () => {
         cy.contains("Move").should("not.exist");
       });
     });
+
+    it.skip("should not allow editing analytics content (#36228)", () => {
+      // dashboard
+      navigationSidebar().findByText(ANALYTICS_COLLECTION_NAME).click();
+      cy.findByTestId("pinned-items")
+        .findByText(METRICS_DASHBOARD_NAME)
+        .scrollIntoView()
+        .click();
+
+      cy.wait("@getDashboard");
```

Comment:
Is there a way to get the dashboard ID programmatically and to then use `visitDashboard()` helper?
It will greatly reduce the time it takes to run this test and, more importantly, it will automatically wait for the dashboard card results to load.

`@getDashboard` still doesn't know anything about dashboard cards.

Reply:
I made a little helper for this and also used it in the flaky test above. Left the @flaky tag on for now though.

Reply:
In that case, maybe don't try to fix it in this PR.
That should be a separate issue and a separate fix.